rust-toolchain

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-# Version updated on 2022-01-14
-channel = "1.60"
```

Comment:
Nit: Make sure the commit message when you merge says that this PR bumps the Rust version number.

We care about this because the Rust version can affect benchmarks and we want to isolate those changes from other things that could affect performance.

Nit (optional): update the version number in its own PR.

Reply:
Made a separate PR